src/scripts/components/stories/pilot-story/components/06-chapter/06-chapter.tsx

```diff
@@ -82,6 +88,11 @@ const ChapterSix: FunctionComponent<Props> = ({chapterIndex}) => {
     storyRef.current?.scrollIntoView({behavior: 'instant'});
   };
 
+  const handleChapterChange = () => {
+    setSelectedChapterIndex(6);
+    scrollToChapterIndex(6);
```

Comment:
@Immo-Be You are still working on this, right? Because the change to chapter 7 does not work here.

Reply:
Indeed :)